common/thorhelper/thorcommon.hpp

```diff
@@ -142,11 +142,11 @@ inline unsigned getCompMethod(unsigned flags)
 }
 
 //MORE: This should be passed as an option ptree instead - see future PRs
-inline unsigned getCompMethod(const char *compStr)
+inline CompressionMethod getCompMethod(const char *compStr)
```

Comment:
The other getCompMethod on line 129 should probably return CompressionMethod too.